project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ package com.gu
 import sbt._
 
 object Dependencies {
-  val identityLibVersion = "3.133"
+  val identityLibVersion = "3.134-SNAPSHOT"
```

Comment:
should we be pushing snapshot libraries?

Reply:
We'll change this before it goes out 👍